examples/basic/main.tf

```diff
@@ -110,7 +123,9 @@ resource "template_file" "user_data" {
 
 ## Provisions basic autoscaling group
 module "example" {
-  source = "github.com/unifio/terraform-aws-asg//group/basic"
+  # Example GitHub source
+  #source = "github.com/unifio/terraform-aws-asg//group/basic"
+  source = "../../group/basic"
```

Comment:
Doesn't actually test what's in the repo until it's committed, which is a pain when you're developing and want to use these to verify.